pkg/controller/elasticsearch/observer/observer.go

```diff
@@ -26,9 +26,9 @@ type Settings struct {
 	Tracer              *apm.Tracer
 }
 
-// defaultObservationInterval is the default interval of observation.
-// if the Elasticsearch cluster is unavailable, the actual interval would be observationInterval + requestTimeout.
-const defaultObservationInterval = 10 * time.Second
+// defaultObservationTimeout is the default timeout for an observation. The observer uses the observation interval as a timeout
+// the default applies if the observation interval is not positive to allow at least one successful observation.
+const defaultObservationTimeout = 10 * time.Second
```

Comment:
Another way to make it explicit that we use the interval for the timeout?
```suggestion
const (
	DefaultInterval = 10 * time.Second
	defaultimeout  = DefaultInterval
)
```

Reply:
I was referring to this bit here https://github.com/elastic/cloud-on-k8s/blob/2033eaa17f277bba0d19eb74ea0c552f6a396069/pkg/controller/elasticsearch/observer/observer.go#L112

Timeout and default interval are independant in principle. I think the reason why by default the interval is used is that we want to avoid a situation where multiple requests are in flight.